Makefile

```diff
@@ -324,6 +330,16 @@ ut combined.coverprofile: vendor/.up-to-date $(FELIX_GO_FILES)
 	@echo Running Go UTs.
 	$(DOCKER_GO_BUILD) ./utils/run-coverage
 
+bin/fv.test: vendor/.up-to-date $(FELIX_GO_FILES)
+	$(DOCKER_GO_BUILD) go test ./fv -c --tags fvtests -o bin/fv.test
```

Comment:
When I added the `k8sfv` directory, I needed to exclude it from some of the infrastructure for running UTs:

```
./utils/run-coverage:11:	      grep -vE '/vendor/|\./proto/|.glide|/k8sfv/' | \
./utils/run-coverage:24:ginkgo -cover -covermode=count -coverpkg=${go_dirs} -r -skipPackage k8sfv
./Makefile:352:	$(DOCKER_GO_BUILD) ginkgo -r -skipPackage k8sfv $(GINKGO_OPTIONS)
./Makefile:357:	$(DOCKER_GO_BUILD) ginkgo watch -r -skipPackage k8sfv $(GINKGO_OPTIONS)
```

Do you need to do that for `fv` as well?